addon-test-support/custom-assertions/tooltip.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Helpers & Modifiers/Modifiers/Tooltip/Assertions"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: { canvas: { hidden: true } }
+  }}
+/>
+
+# Assertions
+
+To help testing tooltip, new `assert` functions will be added. The simple way is to add the following code in `test-helper.ts`
+
+```typescript
+import * as QUnit from 'qunit';
+import registerAssertions from '@upfluence/oss-components/test-support/register-assertions';
+registerAssertions(QUnit.assert);
+```
```

Comment:
nit: IMO, this could go to a higher doc, maybe in the "Getting Started" section as it concerns all of the projects. But we can keep it here as it's the only custom assertions for the moment

Reply:
Agreed !